tools/Environments/DevHome.Environments/ViewModels/ComputeSystemViewModel.cs

```diff
@@ -296,10 +301,11 @@ private void RemoveComputeSystem()
         });
     }
 
-    private void LogFailure(ComputeSystemOperationResult? operationResult)
+    private (string DisplayMessage, string DiagnosticText) LogFailure(ComputeSystemOperationResult? operationResult)
     {
         var messageWhenNull = _stringResource.GetLocalized("EnvironmentOperationFailedUnKnownReasonText");
         var errorMessage = (operationResult != null) ? operationResult.Result.DisplayMessage : messageWhenNull;
+        var diagnosticText = errorMessage;

```

Comment:
line 306 declares and assigns errorMessage. The next line make a copy of errorMessage and assigns it to diagnosticText.  Why?

Reply:
removed this, thanks